### PR TITLE
Prevent title change on export

### PR DESCRIPTION
**Pull request by jbout**
Don't change the title of the qti item in the itembank after exporting it

### CODE REVIEW

File: model/Export/QTIPackedItemExporter.php [hunk 1]
```diff
@@ -105,10 +105,6 @@ public function exportManifest() {
 		
 		if (!is_null($qtiItem)) {
             
-            //tao currently uses the title as the label of imported qti items:
-            $qtiItem->setAttribute('title', $rdfItem->getLabel());
-            $qtiItemService->saveDataItemToRdfItem($qtiItem, $rdfItem);
-
```
Comment (ssipasseuth) [inline model/Export/QTIPackedItemExporter.php]: I understand the issue you are trying to solve. However this fix will make an old issue resurface : if a new item has been created in TAO, its initial title is the rdf label (title is required in the standard). However, if the rdf label changes, the item title in the xml is no longer the same. So when we export that item and reimport it back into tao, the rdf resource will not longer have the right value (but the one set upon creation).

Comment (ssipasseuth) [inline model/Export/QTIPackedItemExporter.php]: The real source of the issue is that we want to provide a way to match the rdf item metadata to the one in the qti item xml. The real fix should be to definitely define our policy regarding this matching. What is your opinion on that ?

**Comment by jbout** [comment]
The current behavior is that exporting an item changes the content of the item, not the content of the zip file. I can understand changing the title in the exported XML, but we should not touch the item in the item-bank.

**Comment by ssipasseuth** [comment]
Actually, I am not even sure we should modify the title at all...

**Pull request merged**